nifpga/tests/test_bitfile.py

```diff
@@ -4,6 +4,7 @@
 import nifpga
 
 BITFILE_ALL_REGISTERS = 'nifpga/tests/allregisters.lvbitx'
+BITFILE_FXP_TYPES = 'nifpga/tests/FXPTypes.lvbitx'
```

Comment:
Go ahead and remove the Bitstream from this bitfile (same way as the other bitfile) since it isn't being used.  That chops down its size a lot.

Reply:
Removed